cmd/kubelet/app/options/options.go

```diff
@@ -552,4 +552,5 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	// Memory Manager Flags
 	fs.StringVar(&c.MemoryManagerPolicy, "memory-manager-policy", c.MemoryManagerPolicy, "Memory Manager policy to use. Possible values: 'None', 'Static'.")
 	fs.Var(&utilflag.ReservedMemoryVar{Value: &c.ReservedMemory}, "reserved-memory", "A comma separated list of memory reservations for NUMA nodes. (e.g. --reserved-memory 0:memory=1Gi,hugepages-1M=2Gi --reserved-memory 1:memory=2Gi). The total sum for each memory type should be equal to the sum of kube-reserved, system-reserved and eviction-threshold. See https://kubernetes.io/docs/tasks/administer-cluster/memory-manager/#reserved-memory-flag for more details.")
+	fs.Int32Var(&c.PLEGChannelCapacity, "pleg-channel-capacity", c.PLEGChannelCapacity, "Capacity of the channel for receiving pod lifecycle events. Defaults to 1000.")
```

Comment:
/hold

We are trying to not add any new CLI flags wherever possible to the kubelet in favour of an updated KubeletConfiguration.

Reply:
@ehashman okay, if the issue could be approved, I will change this, thanks